src/EditorFeatures/Core/Implementation/LineSeparators/AdornmentManager.cs

```diff
@@ -302,6 +316,29 @@ private void UpdateInvalidSpans()
             }
         }
 
+        private NormalizedSpanCollection GetCollapsedSpans()
+        {
+            var elisionSnapshot = _textView.VisualSnapshot as IProjectionSnapshot;
```

Comment:
Is this enough to guarantee that this is an elision buffer?  What if it is some different type of projection buffer?